mcs/class/System.Net.Http/wasm_System.Net.Http.dll.sources

```diff
@@ -1,2 +1,2 @@
-#include legacy.sources
+#include socketshandler.sources
```

Comment:
The idea is to have really only wasm handler here (no sockethandler at all) but I guess that can be cleaned up by @kjpou1

Reply:
We aren't quite there yet, the socket handler cannot be removed at this point.

Reply:
See my question to @kjpou1 - it's that fallback handler that's preventing the sockets handler from being removed.

Reply:
So the `static HttpMessageHandler GetFallback(string message)` is what is preventing that?

Reply:
This method currently calls `new HttpClientHandler ()` - which is only available via the sockets handler.  We could of course return a stubbed-out (`PlatformNotSupportedException`) version of `HttpClientHandler`.